tests/calculators/refl1d/test_refl1d_wrapper.py

```diff
@@ -276,3 +282,137 @@ def test_sld_profile(self):
         p.add_item('Item', 'MyModel')
         assert_almost_equal(p.sld_profile('MyModel')[1][0], 0)
         assert_almost_equal(p.sld_profile('MyModel')[1][-1], 4)
+
+
+def test_get_oversampling():
```

Comment:
I know the numbers of `scale` and `bkg` are not strictly relevant here, but currently used values are completely unphysical, so caution should be taken not to use these values when testing anything to do with the actual reflectivity values.

Reply:
I prefer to keep it that way.  The reason being that when debugging tests it is much easier to deal with "nice" numbers